src/stack.js

```diff
@@ -1,5 +1,42 @@
-'use strict'
-
 export default class Stack {
-  // your code here
+  constructor() {
+    this.size = 0
+    this.elements = []
+  }
+
+  push = (elementToAdd)  => {
+    this.elementToAdd = elementToAdd
+    this.size += 1
+    this.elements.push(elementToAdd)
+  }
+
+  pop = () => {
+    if (this.size == 0) {
+      return null
+    }
+    else {
+      this.size -= 1
+      this.elements.pop()
+    }
+  }
+
+  peek = () => {
+    if (this.size == 0) {
+      return null
+    }
+    else {
+      return this.elements[this.size - 1 ]
+    }
+  }
+
+  isEmpty = () => {
+    if (this.size == 0){
```

Comment:
`this.size === 0 ? true : false`

an `if` that returns a straight boolean is a code smell for me. might not work but check if this passes tests: 

`() => this.size === 0`

because `this.size === 0` should evaluate to a boolean.

also an if that just returns true/false can usually be refactored